src/Qt/VideoCacheThread.cpp

```diff
@@ -74,6 +74,7 @@ namespace openshot
         // Types for storing time durations in whole and fractional microseconds
         using micro_sec = std::chrono::microseconds;
         using double_micro_sec = std::chrono::duration<double, micro_sec::period>;
+        bool should_pause = false;
```

Comment:
Little issue, but for clarity this could be called `should_pause_cache` or `halt_cache` (avoiding the word pause altogether)

Just to distinguish the _cache_ pausing from the _player_ pausing.